apps/calendar_visualizer/ReadMe.md

```
@@ -0,0 +1,35 @@
+Created by: Ctrl-G & Google's Gemini (Flash 3.6 - I think, although it might have been upgraded during this development).
+
+Summary: Calendar Visualization App
+
+Description: Displays a very dense visualization of events from a Google Calendar iCal URL.  Probably only helpful for
+persons who don't have a lot of closely spaced events in their google calendar.  This was produced because I always wanted
+to see the spacial relationship between the current time and my next 'event / meeting'.
+
+The calendar graphing area shows 62 days, one day for each vertical column of LEDs.  Each dot in the vertical column indicates the following time:
+
+<i>Calendar Visualizer - No animation</i>
+<img src = "./calendar_visualizer-DispExplain.gif" >
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)